### PR TITLE
Feat: Added slider

### PR DESCRIPTION
**Pull request by abhijeetekad**
chrome-capture (6)

### CODE REVIEW

**Review by shashidwivedi — APPROVED** [review]
Looks good, can be merged

File: components/slider.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.price-range {
+  display: flex;
+  padding: 2rem 0rem;
+}
+.range {
+  font-size: 18px;
```
Comment (Megha-Joshi) [inline components/slider.css]: use rem instead of px

**Review by Megha-Joshi — APPROVED** [review]
Good to merge.

File: components/slider.css (hunk 1 above)
Comment (abhijeetekad) [reply components/slider.css]: Done. Thanks for pointing this out.

**Pull request merged**